R/temporalBehaviour.R

```diff
@@ -17,42 +17,42 @@ checkInputsTemporalBehaviour <- function(sts, points, product, windThreshold, me
                                          empiricalRMW, tempRes, verbose) {
   # Checking sts input
   stopifnot("no data found" = !missing(sts))
-
+  
```

Comment:
Linting: remove all trailing whitespaces
```suggestion
  
```

Reply:
I did not understand the pb here

Reply:
I think it's fixed now